vllm/executor/ray_gpu_executor.py

```diff
@@ -40,11 +55,7 @@ def _init_executor(self) -> None:
         # Create the parallel GPU workers.
         self._init_workers_ray(placement_group)
 
```

Comment:
nit, but we can add self.use_spmd_worker = VLLM_USE_SPMD_WORKER to avoid keep calling os.getenv

Reply:
Not sure which snippet you were pointing to, but so far in `ray_gpu_executor` the value as retrieved only once and kept in a global variable.